README.md

```diff
@@ -278,25 +288,34 @@ Examples:
 * To use the grouping key `job="directory_cleaner",path="/var/tmp"`, the
   following path will _not_ work:
 
-	  /metrics/job/directory_cleaner/path//var/tmp
-	  
+      /metrics/job/directory_cleaner/path//var/tmp
+      
   Instead, use the base64 URL-safe encoding for the label value and mark it by
   suffixing the label name with `@base64`:
   
-  	  /metrics/job/directory_cleaner/path@base64/L3Zhci90bXA
-	  
+      /metrics/job/directory_cleaner/path@base64/L3Zhci90bXA
+      
   If you are not using a client library that handles the encoding for you, you
   can use encoding tools. For example, there is a command line tool `base64url`
   (Debian package `basez`), which you could combine with `curl` to push from
   the command line in the following way:
   
       echo 'some_metric{foo="bar"} 3.14' | curl --data-binary @- http://pushgateway.example.org:9091/metrics/job/directory_cleaner/path@base64/$(echo -n '/var/tmp' | base64url)
+
+* To use the grouping key `job="example",first_label="",second_label="foobar"`,
```

Comment:
It's not quite clear that the issue here is the empty label. I'd suggest something like "To use a grouping key with an empty label value such as ..."

Reply:
Makes sense, updated.